openIt/ngsild-payloads/authorization.jsonld

```diff
@@ -61,5 +41,33 @@
         "value": "2024-08-30T11:00:00.000Z"
       }
     }
-  ]
+  ],
+  "forGate": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:Gate:123"
+  },
```

Comment:
raccourci de la relation useKey.opens (c'est redondant, mais pas de souci si ça vous simplifie la vie)

Reply:
Frontend, having this link simplifies our life a lot